gophish.go

```diff
@@ -102,18 +111,27 @@ func main() {
 	phishServer := controllers.NewPhishingServer(phishConfig)
 
 	imapMonitor := imap.NewMonitor()
-
-	go adminServer.Start()
-	go phishServer.Start()
-	go imapMonitor.Start()
+	if *mode == "admin" || *mode == "all" {
+		log.Infof("Stating the admin and IMAP server, server mode is %s", *mode)
```

Comment:
Can we please remove this log line? It's not required.

Reply:
Removed